### ISSUE TITLE
Pledge amounts a11y issue/clarity 

### ISSUE THREAD

**Issue by bkardell** [opencollective/opencollective#3586]
**Describe the bug**
This is the bug I accidentally opened in the wrong repo - since it has comments from someone that I cannot 'move' I am just linking here https://github.com/opencollective/opencollective-frontend/issues/5196

**To Reproduce**
Try to choose a pledge amount using assistive technology

**Expected behavior**
It should be clear that this is a choice of predefined amounts or specifiable amounts and you should be able to understand which is selected.

**Comment by znarf** [comment]
Thank you for reporting the issue.

Pledges is not the focus of the team at the moment and it will be difficult to prioritize the issue.

If someone open a PR to improve this, we're happy to look at it and help to get it merged.

**Comment by Betree** [comment]
[We are removing pledges](https://github.com/opencollective/opencollective/issues/6590). We were not happy with how it worked and the usage was not significant enough to justify the maintenance costs. If you'd like to contribute to the discussion about a possible revival of pledges, you can do so at https://github.com/opencollective/opencollective/discussions/6027.
